cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -428,7 +472,7 @@ def is_displayed(self):
             self.collectlogs.is_displayed and
             self.collectlogs.is_active and
             self.title.text == 'Diagnostics Zone "{}" (current)'.format(
-                self.context['object'].description))
+                self.context['object'].appliance.default_zone.description))
```

Comment:
Q: could you explain why default zone ?

Reply:
`self.context['object'].description` gives an error, so I used `appliance.default_zone` as log collection tests use only one zone, which is default.